apache/superset
# chore(Dashboard): Remove unnecessary loading

### SUMMARY
The goal of this PR is unblock the Dashboard skeleton (showing of the filters, tabs, charts containers, etc.) progressively so that the user experience is improved by showing available content sooner rather than later.

The work on this PR was becoming too large, leading to the decision of splitting the changes in smaller chunks. The thread will be updated with all related PRs.

### TESTING INSTRUCTIONS
1. The Dashboard should load normally.
2. A Dashboard with a pre-selected filter value should load successfully.

### ADDITIONAL INFORMATION
